src/metadata.py

```diff
@@ -56,7 +56,7 @@ def get(path: Path) -> types_.Metadata:
         raise InputError(f"Invalid value for name key: {name}, expected a string value")
 
     docs = metadata.get(METADATA_DOCS_KEY)
-    if not isinstance(docs, str | None) or (METADATA_DOCS_KEY in metadata and docs is None):
+    if docs and not isinstance(docs, str) or (METADATA_DOCS_KEY in metadata and docs is None):
```

Comment:
Yes, sorry for that, I might have misunderstood the logic. I had to change this line because it was not passing the tests. My thoughts were to rewrite the condition above and this is the flow of thoughts I had

```
not (isinstance(docs, str) or docs is None)

(not isinstance(docs, str)) and (docs is not None)

(not isinstance(docs, str)) and (docs)
```

I actually tried the two:

```
def new(docs):
    if docs and not isinstance(docs, str):
        return "here"
    else:
        return "there"


def old(docs):
    if not isinstance(docs, str | None):
        return "here"
    else:
        return "there"


docs = 1
assert old(docs) == new(docs)

docs = None
assert old(docs) == new(docs)

docs = ""
assert old(docs) == new(docs)

docs = "test"
assert old(docs) == new(docs)
```

which provide the same output in python 3.10. But maybe this is not the expected behaviour?